Library/Formula/mg3a.rb

```diff
@@ -0,0 +1,40 @@
+class Mg3a < Formula
+  desc "Small Emacs-like editor with UTF-8 support"
+  homepage "http://www.bengtl.net/files/mg3a/"
+  url "http://www.bengtl.net/files/mg3a/mg3a.150703.tar.gz"
+  sha256 "c1e0dec1b2ba1feebc623f5a032653dcab442ef9438449ec556166de504d7a0b"
+
+  conflicts_with "mg", :because => "both install `mg`"
+
+  option "without-emacs-quit", "Use ^U ^X ^C to exit saving"
+  option "with-c-mode", "Include the original C mode"
+  option "with-clike-mode", "Include the C mode that also handles Perl and Java"
+  option "with-python-mode", "Include the Python mode"
+  option "without-dired", "Exclude dired functions"
+  option "without-prefix-region", "Exclude the prefix region mode"
+  option "with-user-modes", "Include the support for user defined modes"
+  option "with-user-macros", "Include the support for user defined macros"
+  option "with-most", "Include c-like and python modes, user modes and user macros"
+  option "with-all", "Include all fancy stuff"
```

Comment:
Hmmm give me a couple of days to think about it...
El 02/08/2015 22:51, "Alex Dunn" <notifications@github.com> escribió:

> In Library/Formula/mg3a.rb
> <https://github.com/Homebrew/homebrew/pull/39706#discussion_r36047719>:
>
> > +  homepage "http://www.bengtl.net/files/mg3a/"
> > +  url "http://www.bengtl.net/files/mg3a/mg3a.150703.tar.gz"
> > +  sha256 "c1e0dec1b2ba1feebc623f5a032653dcab442ef9438449ec556166de504d7a0b"
> > +
> > +  conflicts_with "mg", :because => "both install `mg`"
> > +
> > +  option "without-emacs-quit", "Use ^U ^X ^C to exit saving"
> > +  option "with-c-mode", "Include the original C mode"
> > +  option "with-clike-mode", "Include the C mode that also handles Perl and Java"
> > +  option "with-python-mode", "Include the Python mode"
> > +  option "without-dired", "Exclude dired functions"
> > +  option "without-prefix-region", "Exclude the prefix region mode"
> > +  option "with-user-modes", "Include the support for user defined modes"
> > +  option "with-user-macros", "Include the support for user defined macros"
> > +  option "with-most", "Include c-like and python modes, user modes and user macros"
> > +  option "with-all", "Include all fancy stuff"
>
> @paaguti <https://github.com/paaguti> We might be able to get by with a
> single option with-emacs-style:
>
> if build.with? "emacs-style"
>   args << "-DMOST"else
>   args << "-DALL"end
>
> That should cover most use-cases, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/39706/files#r36047719>.
>